docs/views/layout.html

```diff
@@ -2,8 +2,17 @@
 
 {% block head %}
   {% include "includes/head.html" %}
+  {% if promoMode == 'true' and gtmId %}
+    {% include "includes/tracking_head.html" %}
+  {% endif %}
 {% endblock %}
 
+{% if promoMode == 'true' and gtmId %}
+  {% block body_start %}
```

Comment:
Interesting Nunjucks logic I didn't know about: this block is being written to the page regardless of the `if` above it. In order for the `if` to do what we want it to, you need to the `block` first, then the `if` inside it